IsPalindrom/Program.cs

```diff
@@ -15,4 +12,19 @@ bool isPalindrom (string stringToTest)
     return stringToTest == stringReversed;
   
 }
-System.Console.WriteLine(isPalindrom("racecar"));
\ No newline at end of file
+System.Console.WriteLine(isPalindrom("racecar"));
+
+// approach 2 with Array.Reverse method
+bool isPalindrom2 (string stringToTest)
+{
+    char [] tempCharArray = stringToTest.ToCharArray();
+    // Array.Reverse is a method with void type, means doesn't return anything
+    Array.Reverse(tempCharArray);
```

Comment:
this is a good solution, congrats!

we can make it shorter though:

strings are characters arrays technically, so you dont really need the ToCharArray part if you use them like this.

and you can use Array.Reverse() as an extension method on strings as well, further shortening the code. (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods)

resulting in something like this:

return stringToTest == stringToTest.Reverse();


LINQ (https://docs.microsoft.com/en-us/dotnet/standard/linq/) also has a SequenceEqual method, to check if two collections/sequences are equal. for custom types you would have to implement your own equality comparer method, but for base types (like string) it works out of the box.

so your entire method can be written like this:

return stringToTest.SequenceEqual(stringToTest.Reverse());

Reply:
man this is some nice and simple implementation... thank you!!